cli_tools/gce_vm_image_import/importer/request.go

```diff
@@ -0,0 +1,156 @@
+//  Copyright 2020 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package importer
+
+import (
+	"fmt"
+	"reflect"
+	"regexp"
+	"strings"
+	"time"
+
+	daisyutils "github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/common/utils/daisy"
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/daisycommon"
+	"github.com/GoogleCloudPlatform/compute-image-tools/daisy"
+)
+
+// Flags that are validated.
```

Comment:
Shouldn't these be in imageImportArgs? Flags/args are a CLI concept, not a library concept (that's why you introduced ImageImportRequest)

Reply:
This is a good point. I was hoping to punt on this until we needed the feature.

The main reason: The OVF tool will be constructing, or pre-modifying, the fields that have underscores, so a validation failure means a bug. (For example, `scratch-bucket-gcs-path` is created or validated when the tool starts).

So, at the moment, I found that an indirection layer to customize error messages was adding more code, but without benefit for our current project, or anything that I could think of.